docs/sources/mimir/configure/configure-otel-collector.md

```diff
@@ -10,14 +10,17 @@ weight: 150
 # Configure the OpenTelemetry Collector to write metrics into Mimir
 
 {{% admonition type="note" %}}
-To send OTLP data to Grafana Cloud, refer to [Send data using OpenTelemetry Protocol (OTLP)](https://grafana.com/docs/grafana-cloud/send-data/otlp/send-data-otlp/).
+To send OTel data to Grafana Cloud, refer to [Send data using OpenTelemetry Protocol (OTLP)](https://grafana.com/docs/grafana-cloud/send-data/otlp/send-data-otlp/).
 {{% /admonition %}}
 
 When using the [OpenTelemetry Collector](https://opentelemetry.io/docs/collector/), you can write metrics into Mimir via two options: `prometheusremotewrite` and `otlphttp`.
 
-We recommend using the `prometheusremotewrite` exporter when possible because the remote write ingest path is tested and proven at scale.
+We recommend using each protocol's respective exporter and native Mimir endpoint:
 
-## Remote Write
+- Prometheus data through `prometheusremotewrite`
```

Comment:
```suggestion
- For Prometheus data, `prometheusremotewrite`
```

Reply:
I don't think bullet points need to be worded so close to a full proper sentence, starting it with a prepositional phrase. I do not see other examples of us taking this approach in the docs. 

The word "through" could just be a colon if we want to shorten up

Reply:
Yea, let's make it a colon, it cleans it up a bit.